hack/multi-node/user-data.sample

```diff
@@ -29,7 +29,6 @@ coreos:
         ExecStartPre=/bin/mkdir -p /var/lib/kubelet/pki
         ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/cache/kubelet-pod.uuid
         ExecStart=/usr/lib/coreos/kubelet-wrapper \
-          --allow-privileged \
```

Comment:
Maybe it is worth explaining why this is not needed anymore? In the commit message, for example?

Reply:
Sorry, its a deprecated flag which is no longer available

Reply:
:+1: